gradle.properties

```diff
@@ -1,14 +1,14 @@
 slf4jVersion=1.7.12
 junitVersion=4.12
 jredisVersion=06052013
-jedisVersion=2.7.3
+jedisVersion=2.8.0
 springVersion=4.1.7.RELEASE
 springDataBuildVersion=1.8.0.BUILD-SNAPSHOT
 log4jVersion=1.2.17
-version=1.7.0.BUILD-SNAPSHOT
+version=1.7.0.DATAREDIS-315-SNAPSHOT
```

Comment:
@mp911de I am :smile: . We normally bump the version in the first commit of an issue branch to have dedicated build artifacts. That commit is dropped when merging into master.